pandas/core/interchange/buffer.py

```diff
@@ -49,7 +51,7 @@ def ptr(self) -> int:
         """
         return self._x.__array_interface__["data"][0]
 
-    def __dlpack__(self):
+    def __dlpack__(self) -> Any:
```

Comment:
Could import the appropriate package in an `TYPE_CHECKING` block and then use it here.

Reply:
I think this returns a C PyCapsule. I don't think there's a Python API to type this per se